javascript/chronometer.js

```diff
@@ -1,26 +1,35 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0
+    this.intervalId = 0
   }
   startClick(callback) {
-    // ... your code goes here
+    setInterval(() =>
+      this.currentTime++, 1000)
   }
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60)
   }
   getSeconds() {
-    // ... your code goes here
+    return (this.currentTime - Math.floor(this.getMinutes() * 60))
   }
-  twoDigitsNumber() {
-    // ... your code goes here
```

Comment:
Nice, I think you don't need line 17 if you make use of template strings on line 21 `return `${number}` ;`

Also, if you use the ternary operator that we talked about yesterday then you can get a one-line solution (I'll use a different condition here as well)
```
return number < 10 ?  `0${number}` : `${number}`;